README.md

```diff
@@ -1,4 +1,4 @@
-# medic-couch2pg
+# CHT-couch2pg
```

Comment:
It doesn't feel good "CHT" in uppercase here followed by `-` and then the other part in lowercase and without spaces for a title, instead of `cht-couch2pg` or `CHT - couch2pg`, but is not wrong, just an opinion. Specially for a CLI tool, I would prefer `cht-couch2pg` (all lower case), and checking one of the examples @garethbowen mention in the ticket:

> ... Medic Gateway -> CHT Gateway

My be worth to follow a pattern for all the renaming PRs, so, beside the decision of renaming ~medic-~ → _cht-_ in the CLI names and project URLs, do we discussed how to rename the product names?

Reply:
I interpreted the section as a "project name", so that's why I used caps. 
![image](https://user-images.githubusercontent.com/35681649/119685831-62d84800-be4e-11eb-9424-41b13b2bebe7.png)

I agree the cli names should be lowercase, and it's how I changed the other instances:
![image](https://user-images.githubusercontent.com/35681649/119686015-95824080-be4e-11eb-84b1-681f9b58743f.png)

Reply:
Maybe just remove the hyphen from the title?